master/buildbot/test/unit/www_hooks_bitbucket_json_data.py

```diff
@@ -0,0 +1,811 @@
+# This file is part of Buildbot.  Buildbot is free software: you can
+# redistribute it and/or modify it under the terms of the GNU General Public
+# License as published by the Free Software Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright Buildbot Team Members
+# Copyright Manba Team
+
+
+gitJsonPayload = """{
```

Comment:
Sorry for not being clear enough.
I think it is better to use fixture directory for storing big json fixtures:

    gitJsonPayload  = open(util.sibpath(__file__, os.path.join("fixtures", "git_json_payload.json")).read()